nifi-extension-bundles/nifi-poi-bundle/nifi-poi-services/src/main/java/org/apache/nifi/excel/InputFileType.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.excel;
+
+import org.apache.nifi.components.DescribedValue;
+
+public enum InputFileType implements DescribedValue {
+    XLS("XLS", "An XLS file"),
+    XLSX("XLSX", "An XLSX file");
```

Comment:
```suggestion
    XLS("XLS", "HSSF '97(-2007) file format"),
    XLSX("XLSX", ""XSSF 2007 OOXML file format"");
```

Reply:
The `HSSF` acronym should not be used in user-facing labels and documentation. It is an acronym internal to the Apache POI project, and a pejorative one.

Reply:
Yes, including the years in the description is helpful.

Reply:
Good point, changed